presentation/scripts/lab2-1.sh

```diff
@@ -2,33 +2,35 @@
 
 source ./common.sh
 
-SERVICE_PORT=$(kubectl get svc hello-world -ojson | grep nodePort | sed "s/.*: *\([0-9]*\).*/\1/g")
+SERVICE_PORT=$(kubectl get svc guestbook -ojson | grep nodePort | sed "s/.*: *\([0-9]*\).*/\1/g")
 WORKER_IP=$(bx cs workers ${CLUSTER_NAME} --json  | grep publicIP | sed "s/.*\"\([0-9].*\)\".*/\1/g" )
 
-HELLO_CURL=${WORKER_IP}:${SERVICE_PORT}
+GUESTBOOK_CURL=${WORKER_IP}:${SERVICE_PORT}
 
 # make sure the service works
 comment "We're starting with the same output as the previous lab"
-doit curl -s ${HELLO_CURL}
+doit curl -s ${GUESTBOOK_CURL}
 
 comment we have the pods
-doit kubectl get pods -l run=hello-world
+doit kubectl get pods -l run=guestbook
 
 # these pods come from the deployment
 # kubectl get deployment ${DEPLOYMENT_NAME}
 
 comment --nolf "You should start this watch in a separate terminal"
-comment "    watch -d -n 0.2 curl -s ${HELLO_CURL}"
-comment --pauseafter "Press ENTER when ready"
+comment --nocr --nohash --pauseafter "How many replicas? "
+comment --nohash $replicas
```

Comment:
this seems a weird change, bad rebase with the below content?

Reply:
yea, what happened to the watch? Now we have two "How many replicas?"  And you don't need the "read -p" version since the one above will do it for you - choosing a random number for the presenter.

Reply:
Fixed.